Unix/http/httpclientauth.c

```diff
@@ -1436,23 +1461,22 @@ HttpClient_EncryptData(_In_ HttpClient_SR_SocketData * handler, _Out_ Page **pHe
         iov[1].type = GSS_IOV_BUFFER_TYPE_DATA;
         iov[1].buffer.length = original_content_len;
         iov[1].buffer.value  = poriginal_data;
+        iov[2].type = 9 /*GSS_IOV_BUFFER_TYPE_PADDING*/  | GSS_IOV_BUFFER_FLAG_ALLOCATE;
```

Comment:
I'm not sure how this would not build unless `GSS_IOV_BUFFER_FLAG_ALLOCATE` was not defined. Type 9 (presumably `GSS_IOV_BUFFER_TYPE_PADDING`, which must not be defined or Bruce would have used it) ored with `GSS_IOV_BUFFER_FLAG_ALLOCATE`.

What am I missing?